README.md

```diff
@@ -32,9 +32,10 @@ sampleRate    : 16000  // audio sample rate
 threshold     : 0.5    // silence threshold (rec only)
 verbose       : false  // log info to the console
 recordProgram : 'rec'  // Defaults to 'rec' - also supports 'arecord' and 'sox'
+device        : null   // recording device, applicable for arecord only (e.g.: 'plughw:1')
```

Comment:
Setting this will also work for `sox` because it will set the `AUDIODEV` environment variable.